Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/CertificatePersonIdentifier.kt

```diff
@@ -8,7 +10,7 @@ import de.rki.coronawarnapp.vaccination.core.qrcode.VaccinationCertificateQRCode
 import org.joda.time.LocalDate
 import timber.log.Timber
 
-data class VaccinatedPersonIdentifier(
+data class CertificatePersonIdentifier(
```

Comment:
All DCC certs use the same name/DOB pattern as unique identifier, as we have to combine a few different types (per person) in a future sprint, small alignment here already.